package.json

```diff
@@ -74,22 +76,34 @@
     "ignore": [],
     "delay": "2500"
   },
-  "jest": {
-    "moduleFileExtensions": [
-      "ts",
-      "js"
-    ],
-    "transform": {
-      "\\.ts$": "ts-jest"
-    },
-    "moduleDirectories": [
-      "node_modules",
-      "lib"
-    ],
-    "testEnvironmentOptions": {
-      "url": "http://localhost/"
-    },
-    "testRegex": "\\.spec\\.ts$"
+  "vitest": {
+    "test": {
+      "globals": true,
+      "include": [
+        "**/*.spec.ts"
+      ],
+      "environment": "jsdom",
+      "coverage": {
+        "enabled": true,
+        "reportsDirectory": "./coverage",
+        "reporter": [
+          "text-summary"
+        ]
+      },
+      "setupFiles": [],
+      "resolveSnapshotPath": {
+        "lib": "./src"
+      },
+      "alias": {
+        "lib": "./lib",
+        "@": "./src"
+      },
```

Comment:
We don't use these aliases in the source code. I did experiment with the `@` alias, but there is an issue with this as the tsc compiler doesn't rewrite the paths and it would involve more work.